src/js/tracks/text-track-settings.js

```diff
@@ -352,18 +358,18 @@ class TextTrackSettings extends Component {
     const values = this.getValues();
 
     try {
-      if (Object.getOwnPropertyNames(values).length > 0) {
-        window.localStorage.setItem('vjs-text-track-settings', JSON.stringify(values));
+      if (values) {
```

Comment:
Good catch. That was from some earlier work where `getValues()` was not always returning an object.